babel.config.js

```diff
@@ -1,6 +1,6 @@
-module.exports = function(api) {
+export default function (api) {
```

Comment:
This config is not needed

Reply:
This file has been deleted from the project